libraries/chain/controller.cpp

```diff
@@ -3111,22 +3123,19 @@ struct controller_impl {
 
    // called from net threads and controller's thread pool
    vote_status process_vote_message( const vote_message& vote ) {
-      auto aggregate_vote = [&vote](auto& forkdb) -> std::pair<vote_status, std::optional<uint32_t>> {
-         auto bsp = forkdb.get_block(vote.proposal_id);
-         if (bsp)
-            return bsp->aggregate_vote(vote);
-         return {vote_status::unknown_block, {}};
+      auto aggregate_vote = [&vote](auto& forkdb) -> vote_status {
+          auto bsp = forkdb.get_block(vote.proposal_id);
+          if (bsp) {
+             return bsp->aggregate_vote(vote);
+          }
+          return vote_status::unknown_block;
       };
       // TODO: https://github.com/AntelopeIO/leap/issues/2057
       // TODO: Do not aggregate votes on block_state if in legacy block fork_db
```

Comment:
I think we can remove this comment, right?

Reply:
What I mean, is that with the current code in your PR we do not aggregate votes on block_states that are in legacy block fork_db because `aggregate_vote_legacy` is a no-op. So it seems that the comment specifies a TODO which is already done.

Reply:
Wehn #2057 is done there will be two fork database active at once. When the change is made this is a reminder that this code needs to still not aggregate the votes into legacy. I think it best to leave for now and remove when #2057 is worked.

Reply:
Oh yes this makes total sense, thanks.